src/server/utils/getChatCompletion.ts

```diff
@@ -25,10 +27,12 @@ export async function getChatCompletion(
     output: Prisma.JsonNull,
     errorMessage: null,
     statusCode: response.status,
+    timeToComplete: 0
   };
 
   try {
     resp.output = await response.json();
+    resp.timeToComplete = Date.now() - start;
```

Comment:
nit: should probably be before awaiting the json because that could take an unknown amount of time if the thread loses priority.

Reply:
Sounds good